server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -324,6 +325,8 @@ private void moveRootTabletToRootTable(IZooReaderWriter zoo) throws Exception {
   @SuppressFBWarnings(value = "DM_EXIT",
       justification = "TODO probably not the best to call System.exit here")
   private void upgradeZookeeper() {
+    ServerContext context = getContext();
```

Comment:
no good reason

tl;dr

I think this was just incidental to the editing I did. In a lot of places, they were using "context" to refer to the instance variable, but now needed to call getContext() to get it from the super class. To make the diff smaller, I stowed those in a local variable for reuse throughout the method. I think this one was a case where it already was using getContext() and I switched it to use the local variable instead... probably because I thought I undoing my own edit to make it use the method...without realizing it was already using the method before I edited the code.